sql/src/test/scala/org/apache/sedona/sql/predicateTestScala.scala

```diff
@@ -140,6 +140,37 @@ class predicateTestScala extends TestBaseScala {
       assert(equaldf.count() == 0, s"Expected 0 value but got ${equaldf.count()}")
 
     }
+
+    it("Passed ST_Equals for ST_LineFromText and ST_Polygon") {
```

Comment:
Please create a simple test case. Not sure why this has anything to do with polygons.

Reply:
Removed this test case and added a simple test case in constructorTestScala.scala file.